src/formatDate.js

```diff
@@ -50,7 +50,77 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dayParts = date.split(fromFormat[fromFormat.length - 1]);
+
+  let day = '';
+  let month = '';
+  let year = '';
+  const devider = toFormat[toFormat.length - 1];
+  const formatDay = 'DD';
+  const formatMonth = 'MM';
+  const formatYearFourChars = 'YYYY';
+  const formatYearTwoChars = 'YY';
+  const toTwentyFirstCentury = 2000;
+  const toTwentiethCentury = 1900;
+  const limitYearForCentury = 30;
```

Comment:
Static constants are best stored outside the function so that they are not recreated during multiple calls to the function.